src/help.ts

```diff
@@ -76,4 +91,63 @@ export const FUNCTION_HELP_DEFINITIONS: Record<string, string[]> = {
     'Function that compares two `decimal` operands and evaluates to `true` if the left operand is numerically greater than or equal to the right operand.' +
       '\n\nhttps://docs.cedarpolicy.com/policies/syntax-operators.html#function-greaterThanOrEqual',
   ],
+  // Datetime extension functions
+  datetime: [
+    'datetime(String): datetime',
+    '*(experimental)* Function that constructs a `datetime` value from a string in one of the forms:' +
```

Comment:
The types are marked as experimental but they were stabilized in v4.4.0

Reply:
nice catch - I forgot to update after 4.3.2